app/views/play/CampaignView.coffee

```diff
@@ -611,6 +611,8 @@ module.exports = class CampaignView extends RootView
           storage.save "pointed-out-promotion", timesPointedOutPromotion + 1
     @applyCampaignStyles()
     @testParticles()
+    # just hiding it from play view for now. drift state is managed in vue so not possible to change its state here.
```

Comment:
We can access Vuex even though this page is not using Vue.

Reply:
How can we do that? Also, how updateDriftConfiguration is triggered? It seems like routeUpdated got called. Is it because of `@trigger 'did-load-route'` in Router.coffee?